handlers/features.go

```diff
@@ -93,3 +92,71 @@ func (oh *featureHandler) GetFeatureByUuid(w http.ResponseWriter, r *http.Reques
 	w.WriteHeader(http.StatusOK)
 	json.NewEncoder(w).Encode(workspaceFeature)
 }
+
+func (oh *featureHandler) CreateOrEditFeaturePhase(w http.ResponseWriter, r *http.Request) {
+	ctx := r.Context()
+	pubKeyFromAuth, _ := ctx.Value(auth.ContextKey).(string)
+	if pubKeyFromAuth == "" {
+		fmt.Println("no pubkey from auth")
+		w.WriteHeader(http.StatusUnauthorized)
+		return
+	}
+
+	newPhase := db.FeaturePhase{}
+	decoder := json.NewDecoder(r.Body)
+	err := decoder.Decode(&newPhase)
+	if err != nil {
+		w.WriteHeader(http.StatusBadRequest)
+		fmt.Fprintf(w, "Error decoding request body: %v", err)
+		return
+	}
+
+	newPhase.CreatedBy = pubKeyFromAuth
+
+	phase, err := oh.db.CreateOrEditFeaturePhase(newPhase)
+	if err != nil {
```

Comment:
@MahtabBukhari you didn't specify the Phase Uuid which will make the code always update, when you try to create more than one phase, and also didn't account for UpdatedBY, there should be a conditional to check if CreatedBY exists, if it exists the UpdatedBy should  be updated `UpdatedBy = &now`.

Reply:
@elraphty Could you please review it 

![image](https://github.com/stakwork/sphinx-tribes/assets/98140673/764de862-a840-429b-b38d-a33ca6e9bc14)

I have updated it in conditional bases that I think good

Reply:
you didn't create a UUId, you to do something like this `features.Uuid = xid.New().String()`, and you don't need the updatedBy and createdBy in that condition. this works

```sh
if features.Uuid == "" {
		features.Uuid = xid.New().String()
	}
```

Reply:
> you didn't create a UUId, you to do something like this `features.Uuid = xid.New().String()`, and you don't need the updatedBy and createdBy in that condition. this works
> 
> ```shell
> if features.Uuid == "" {
> 		features.Uuid = xid.New().String()
> 	}
> ```

Could you please take a look it again 

![image](https://github.com/stakwork/sphinx-tribes/assets/98140673/1344d5a6-46b7-4953-aab4-b1ae343f9cda)

is that's good?